### PR TITLE
Better usage

### LINKED ISSUE #106: Add better usage section
Add better usage section to readme

### LINKED ISSUE #116: Better documentation

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**
fixes #116
fixes #106

### CODE REVIEW

**Comment by JakeRoggenbuck** [comment]
We need to squash and merge this

**Review by Shuzhengz — APPROVED** [review]
lmao thats alotta commits

File: README.md [hunk 1]
```diff
@@ -81,21 +83,199 @@ sudo acs run
...
 
-### Get
+<br>
+
+## Get
+
+### Flags
+`--raw` is the only used flag for the `get` command.
+
+### Subcommands
```
Comment (Shuzhengz) [inline README.md:93]: hmm maybe it’s better to put detailed instructions in another markdown file instead of the readme, keeping it relatively short

Comment (JakeRoggenbuck) [reply README.md:93]: Yea, thats the idea with the hide away stuff, but maybe another md doc would be good

**Comment by JakeRoggenbuck** [comment]
Lets move it if it seems too cluttered, but I think it might be good for now

**Pull request merged**